pkg/windowsdesktop/sfx/Microsoft.WindowsDesktop.App.Bundle.bundleproj

```diff
@@ -17,24 +17,29 @@
   </ItemGroup>
 
   <!-- Obtain the .NET Core Runtime installers from the VS insertion packages. -->
-  <ItemGroup>
-    <InsertionPackageRID Include="@(RestoreBuildRID)" Arch="$([System.String]::new('%(Identity)').Replace('win-', ''))" />
+  <Target Name="CollectVSInsertionPackageDownloads"
+          BeforeTargets="CollectPackageReferences">
 
-    <NETCoreAppInstallerMsiRID Include="@(InsertionPackageRID)" Name="runtime" InsertionName="SharedFramework" />
-    <NETCoreAppInstallerMsiRID Include="@(InsertionPackageRID)" Name="hostfxr" InsertionName="HostFXR" />
-    <NETCoreAppInstallerMsiRID Include="@(InsertionPackageRID)" Name="host" InsertionName="SharedHost" />
+    <ItemGroup>
+      <InsertionPackageRID Include="@(RestoreBuildRID)" Arch="$([System.String]::new('%(Identity)').Replace('win-', ''))" Condition="'%(RestoreBuildRID.RuntimeMsiExists)' == 'true'"/>
```

Comment:
I would chop this for readability:

```suggestion
      <InsertionPackageRID
        Include="@(RestoreBuildRID)"
        Arch="$([System.String]::new('%(Identity)').Replace('win-', ''))"
        Condition="'%(RestoreBuildRID.RuntimeMsiExists)' == 'true'" />
```

Reply:
yeah, I was considering that, but before wrapping the group in target it wasn't as bad. I'll update it